CHANGELOG.md

```diff
@@ -1,5 +1,69 @@
 # Chef Server Changelog
 
+## 12.4.0 (2016-01-27)
+
+### Components
+
+#### Updated
+* openssl (1.0.1p -> 1.0.1q)
+* knife-opc (528be923 -> 0b8fa0fa)
+* ohai (f1e35bf1 -> 81f1c968)
+* chef (2fe875ce -> 3f3fbc8f)
+
+#### New
+* rest-client (1.8.0)
```

Comment:
I'm pretty sure it's a gem as well. This was from the `omnibus changelog generator` script. Are the components supposed to be only system libraries?